source/versioning/index.html.md.erb

```diff
@@ -1,9 +1,9 @@
 ---
-title: Versioning
-weight: 130
+title: Stay up to date
+weight: 190
 ---
 
-# Versioning
+# Stay up to date
```

Comment:
This is a much clearer title. if we're making it a more general thing about staying up to date should we include other things, e.g. subscribe to the statusio page? Or should we make it about staying up to date with the API?

Reply:
I think specifying that it is keep the API up to date is the way forward for now. We can iterate further as ever.